sw/device/silicon_creator/rom/BUILD

```diff
@@ -176,6 +187,10 @@ opentitan_binary(
     linker_script = ":linker_script",
     deps = [
         ":mask_rom_lib",
+        # We depend on rom_isrs here rather than in mask_rom_lib so that the
+        # `rom_exception_handler` symbol remains unresolved and can be supplied
+        # by tests such as `rom_epmp_test`.
+        ":rom_isrs",
```

Comment:
I had to move the `:rom_isrs` dependency here and adjust the symbol name to `rom_exception_handler` to allow the `rom_epmp_entry` test to pass.  That test supplies its own `rom_exception_handler` and we don't want the ECC handler to clash with it.

Reply:
SGTM